From 6887e996ad771d22659c0eb8cbca94c936fdb10a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ren=C3=A9=20=27Necoro=27=20Neumann?= <gentoo@necoro.eu>
Date: Sun, 15 Sep 2013 14:02:32 +0200
Subject: Fix the config for Gentoo's default needs.

    * Set pathes
    * Set rights.type to 'owner_only' to model pre-0.8 behavior
      (None means 'everyone is allowed everything')
    * Remove 'public_users' and 'private_users' as they are obsolete.
---
 config  | 13 ++++---------
 logging |  2 +-
 2 files changed, 5 insertions(+), 10 deletions(-)

diff --git a/config b/config
index 453a45e..10c1595 100644
--- a/logging
+++ b/logging
@@ -45,7 +45,7 @@ formatter = simple
 [handler_file]
 # File handler
 class = FileHandler
-args = ('/var/log/radicale',)
+args = ('/var/log/radicale/radicale.log',)
 formatter = full
 
 
-- 
1.8.1.5

